src/crypto/internal/boring/sig/sig_other.go

```diff
@@ -0,0 +1,17 @@
+// Package sig stubs internal/boringcrypto/sig
```

Comment:
Nit: should be `crypto/internal/boring/sig`

Reply:
Fixed and repushed.